test/elixir_google_scraper_web/controllers/api/v1/keyword_controller_test.exs

```diff
@@ -1,23 +1,85 @@
 defmodule ElixirGoogleScraperWeb.Api.V1.KeywordControllerTest do
   use ElixirGoogleScraperWeb.ConnCase, async: true
+  use ElixirGoogleScraperWeb.ApiCase, async: true
```

Comment:
Having these 2 looks a bit weird. I think we have 2 option here

1. Use `ApiCase` -- Remove `ConnCase` from here and add everything we need in `ApiCase`
2. Use `ConnCase ` -- Create another module in test support. Maybe we can call `AuthenticationHelper` and move all helpers there.

```elixir
def register_and_log_in_user ...
def login ...
def authenticated_api_conn ...
```

Then we include this helper to conn case

```elixir
import ElixirGoogleScraperWeb.AuthenticationHelper
```